.github/workflows/release_ping.yml

```diff
@@ -0,0 +1,37 @@
+# WIP: Workflow for sending a Slack notification when a new release is published.
+name: New Release notification
+on:
+  release:
+    types: [published]
+
+jobs:
+  send_message:
+    name: Send Slack message
+    runs-on: ubuntu-latest
+    env:
+      release_url: ${{ github.event.release.html_url }}
+      release_name: ${{ github.event.release.name }}
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+    steps:
+      - name: Send notification for new release
+        if: github.event.action == 'published'
+        id: slack
+        uses: slackapi/slack-github-action@v1.24.0
+        env:
+          release_status: ${{ github.event.release.draft && ' draft ' || ' ' }}
+          release_icon: ${{ github.event.release.draft && ':pr-draft:' || ':pull-request:' }}
```

Comment:
This only runs when the release is published, so the draft branch will not be exercised. Also the Slack emojis for `:pr-draft:` and `:pull-request:` are not quite appropriate for releases. They look like this:

<img width="584" alt="Screenshot 2024-01-26 at 10 08 02 AM" src="https://github.com/Automattic/stream-builder/assets/16580576/415fdcb8-6cd0-485b-988e-52e0cb39e6a4">

<img width="584" alt="Screenshot 2024-01-26 at 10 08 48 AM" src="https://github.com/Automattic/stream-builder/assets/16580576/e6c3c425-40d5-4ce2-84ee-ebd747a213f5">